packages/widgets/src/components/NotificationCard/NotificationCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { useTheme } from '@emotion/react';
+import { Badge, CardBase, Icon, Indicator, Typography } from '@ssa-ui-kit/core';
+import {
+  notifyBadge,
+  notifyCard,
+  notifyTitle,
+  notifyText,
+  notifyTime,
+} from './style';
+import { INotificationCardProps } from './types';
+import { timeAgo } from '@ssa-ui-kit/utils';
+
+const NotificationCard = ({
```

Comment:
Could we use export const NotificationCard to avoid yet another time to export default as NotificationCard and simplify imports?

Reply:
done